charts/background-worker/values.yaml

```diff
@@ -9,12 +9,11 @@ resources:
     cpu: "100m"
   limits:
     memory: "512Mi"
-# TODO(elland): Create issue for a metrics endpoint
+# FUTUREWORK: Implement metrics
```

Comment:
we can do this in a separate PR, but we should make sure we won't forget and have a ticket:

https://wearezeta.atlassian.net/browse/FS-2020

btw, which ticket is this PR about?

Reply:
It is this: https://wearezeta.atlassian.net/browse/FS-1940
It was in the title, also added it to description now.

Reply:
I added a `parent of` relation, maybe that's all that was needed :crossed_fingers: